src/App.css

```diff
@@ -1,66 +0,0 @@
-.App {
-  text-align: center;
-  padding:20px;
-  width:100vh;
-  height: 100vh;
-}
-
-.App-logo {
-  height: 5vmin;
-  background-color: cornflowerblue;
-  font-size:50px;
-  justify-content: center;
-}
-
-.Card-Container {
-  background-color: #366a72;
-  display: flex;
-  flex-wrap: wrap;
-  width: 100vh;
-
-}
-
-.App-link {
-  color: #61dafb;
-}
-h2{
-  font-size: 24px;
-  color:black;
-  margin:20px;
-  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
-}
-h3{
-  font-size:16px;
-  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
```

Comment:
font-family can be set at a higher level

Reply:
Okay, its in the body class in index.css